sections/kubecontext.zsh

```diff
@@ -36,8 +36,27 @@ spaceship_kubecontext() {
     [[ -n $kube_namespace && "$kube_namespace" != "default" ]] && kube_context="$kube_context ($kube_namespace)"
   fi
 
+  local section_color
+
+  # Apply custom color to section if $kube_context matches a pattern defined in SPACESHIP_KUBECONTEXT_COLORGROUPS array.
+  # See Options.md for usage example.
+  local len=${#SPACESHIP_KUBECONTEXT_COLORGROUPS[@]}
```

Comment:
Would it be possible to have this variable in the top of this file, where we have all default values? Would this syntax be valid? I usually look in that part of the file to see what I can configure before I dig into Options.md...

```
SPACESHIP_KUBECONTEXT_COLORGROUPS="${SPACESHIP_KUBECONTEXT_COLORGROUPS=()}"
```

Reply:
Except that it should be without quotes, otherwise it will take it as a single value.
```zsh
$ var1=(aa bbb ccc)
$ var2="(${var1=()})"
$ echo $var1
aa bbb ccc
$ echo $var2[@]
(aa bbb ccc)
$ echo $#var1
3
$ echo $#var2
12
```